### PR TITLE
Fix multimodal textbox custom components

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

Closes: #8718

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

**Comment by abidlabs** [comment]
Thanks @freddyaboulton! Could we add a test to ensure that all of our core components can be used as templates for custom components? Would be helpful as we add new components (e.g. TimeRange)

**Comment by freddyaboulton** [comment]
Modified the tests @abidlabs ! Thanks for the suggestion

File: test/test_gradio_component_cli.py [hunk 1]
```diff
@@ -4,39 +4,31 @@
...
-        "FileExplorer",
-        "Model3D",
-    ],
+    core,
 )
 def test_template_override_component(template, tmp_path):
+    """When you add a new component this test will likely fail locally
+    because the js files have not been moved to the _frontend_code directory.
+
+    Just build the python package to move the latest state of the js directory to _frontend_code.
```
Comment (abidlabs) [inline test/test_gradio_component_cli.py:30]: What does "build the python package" mean? Do you mean build the frontend?

**Review by abidlabs — APPROVED** [review]
lgtm thanks for updating the test @freddyaboulton! just one question above

File: test/test_gradio_component_cli.py (hunk 1 above)
Comment (freddyaboulton) [reply test/test_gradio_component_cli.py:30]: Need to build the wheel so that a script to copy the js files gets run. Let me update comment!

**Pull request merged**